wsl/accessibility/ubuntu-wsl-doc-images.md

```diff
@@ -41,123 +41,126 @@ explain what it is and what it does:
 
 # Table of contents
 
+- [Introduction](#introduction)
+- [Image accessibility and alt text](#image-accessibility-and-alt-text)
+- [Table of contents](#table-of-contents)
 - [Tutorials](#tutorials)
-    - [Working with Visual Studio Code](#working-with-visual-studio-code) 
-    - [Windows and Ubuntu Interoperability](#windows-and-ubuntu-interoperability)
-    - [Run a .NET Echo Bot as a systemd service on Ubuntu WSL](#run-a-.NET-Echo-Bot-as-a-systemd-service-on-Ubuntu-WSL)
-    - [Enabling GPU acceleration with the NVIDIA CUDA platform](#enabling-GPU-acceleration-with-the-NVIDIA-CUDA-platform)
-    - [Use WSL for data science and engineering](#use-WSL-for-data-science-and-engineering)
+  - [Working with Visual Studio Code](#working-with-visual-studio-code)
+  - [Windows and Ubuntu Interoperability](#windows-and-ubuntu-interoperability)
+  - [Run a .NET Echo Bot as a systemd service on Ubuntu WSL](#run-a-net-echo-bot-as-a-systemd-service-on-ubuntu-wsl)
+  - [Enabling GPU acceleration with the NVIDIA CUDA platform](#enabling-gpu-acceleration-with-the-nvidia-cuda-platform)
+  - [Use WSL for data science and engineering](#use-wsl-for-data-science-and-engineering)
 - [Howtos](#howtos)
-    - [Install Ubuntu on WSL2](#install-Ubuntu-on-WSL2)
+  - [Install Ubuntu on WSL2](#install-ubuntu-on-wsl2)
 
 # Tutorials
 
 ## Working with Visual Studio Code
 
 [Link to full documentation page](https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/tutorials/vscode/) 
 
-![|624x483](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/vscode/msstore.png?raw=true)
+![Installation page for Visual Studio Code on the Microsoft Store](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/vscode/msstore.png?raw=true)
 
 As an example, alt text for the image above could be: `Install page for Visual Studio Code on the Microsoft Store`.
 
-![|624x353](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/vscode/download-vs-code.png?raw=true)
+![Visual Studio Code download page showing download options for Windows, Linux (.deb and .rpm packages), and Mac](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/vscode/download-vs-code.png?raw=true)
 
-![|624x492](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/vscode/aditional-tasks.png?raw=true)
+![Visual Studio Code's installation "Additional Tasks" Steps with the "Add to PATH" and "Register Code as an editor for supported file types" options checked](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/vscode/aditional-tasks.png?raw=true)
```

Comment:
I want to note here that it is sometimes advised to avoid including quotes within alt text as it can create issues in the html but I did not observe any issues during testing in the browser or with a screen reader. Where possible, however, we should use them consistently, i.e., don't sometimes use them and other times not use them, and don't sometimes use single quotes and other times double quotes.

Reply:
Hi @edibotopic, could you give more insight on what you mean by "don't sometimes use them and other times not use them" in relation to consistency in using quotes?

Reply:
I just meant that the most important thing is to be consistent in whichever choice you make. Originally, I found that you were adding quotes around a type of term in one case but not another, and that sometimes you would use single quotes and other times double. In general, it's good to be consistent when making such a choice. The latest version does not have these issues.